### PR TITLE
Update rustfmt

### PR DESCRIPTION
**Pull request by sgrif**
Old versions of rustfmt-nightly are no longer compiling since they don't
disable default features of serde, and the default features of serde
require Rust 1.26. Unfortunately, the rustfmt that ships with 1.26 is
super buggy, but all previous stable versions error on lines they can't
reformat that are >=100 characters, so we'll have to work around them.

### CODE REVIEW

**Comment by killercup** [comment]
Wasn’t 1.26.1 just released with fixes to rustfmt?

Sean Griffin <notifications@github.com> schrieb am Mi. 30. Mai 2018 um
19:42:

**Comment by sgrif** [comment]
This is using 1.26.1

**Comment by sgrif** [comment]
@killercup  https://github.com/diesel-rs/diesel/pull/1743/files#diff-354f30a63fb0907d4ad57269548329e3R49

File: diesel/src/pg/connection/result.rs [hunk 1]
```diff
@@ -83,12 +83,11 @@ impl PgResult {
 
     pub fn is_null(&self, row_idx: usize, col_idx: usize) -> bool {
         unsafe {
-            0
```
Comment (Eijebong) [inline diesel/src/pg/connection/result.rs]: :laughing:

File: diesel/src/pg/types/network_address.rs [hunk 2]
```diff
@@ -185,12 +195,38 @@ fn some_v4address_from_sql() {
...
-            let test_address = IpNetwork::V6(Ipv6Network::new(Ipv6Addr::new(0, 0, 0, 0, 0, 0, 0, 1), 64).unwrap());
+            let test_address = IpNetwork::V6(
+                Ipv6Network::new(Ipv6Addr::new(0, 0, 0, 0, 0, 0, 0, 1), 64).unwrap(),
+            );
             ToSql::<$ty, Pg>::to_sql(&test_address, &mut bytes).unwrap();
-            assert_eq!(bytes, vec![PGSQL_AF_INET6, 64, $net_type, 16, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1]);
-        }
+            assert_eq!(
+                bytes,
+                vec![
```
Comment (Eijebong) [inline diesel/src/pg/types/network_address.rs]: Can we not do that ? :p

**Pull request merged**